### PR TITLE
Add alternative orthophotos for Norway

### PR DESCRIPTION
**Pull request by NKAmapper**
This alternative orthophoto tile service is getting more frequent updates than the existing service, however it is provided down to zoom 19 only vs zoom 21 for the other service.

[Syncs with JOSM](https://josm.openstreetmap.de/wiki/Maps/Norway#NorwayOrthophotomorerecentlesszoom)

### CODE REVIEW

**Comment by grischard** [comment]
Thank you! Did you test it in the editor? I wonder what FORMAT=image/jpgpng ends up producing.

**Comment by NKAmapper** [comment]
It works in [preview.ideditor.com/master](http://preview.ideditor.com/master/).

I am not familiar with this format at all. According to [ArcGIS documentation](https://developers.arcgis.com/rest/services-reference/export-image.htm) _"this format returns a JPG if there are no transparent pixels in the requested extent; otherwise, it returns a PNG"_.

File: sources/europe/no/Norway-ortho2.geojson [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    "properties": {
+        "best": true, 
+        "type": "tms",
+        "name": "Norway Orthophoto (more recent, less zoom)",
+        "id": "geovekst-nib2",
+        "country_code": "NO",
+        "description": "Most recent Norwegian orthophotos (max zoom 19), courtesy of Geovekst and Kartverket.",
+        "url": "https://opencache{switch:,2,3}.statkart.no/gatekeeper/gk/gk.open_nib_web_mercator_wmts_v2?SERVICE=WMTS&REQUEST=GetTile&VERSION=1.0.0&LAYER=Nibcache_web_mercator_v2&STYLE=default&FORMAT=image/jpgpng&tileMatrixSet=default028mm&tileMatrix={zoom}&tileRow={y}&tileCol={x}",
+        "icon": "https://register.geonorge.no/data/organizations/_L_norgeibilder96x96.png",
+        "max_zoom": 21,
```
Comment (don-vip) [inline sources/europe/no/Norway-ortho2.geojson:12]: the max-zoom is wrong no? @NKAmapper @grischard

Comment (NKAmapper) [reply sources/europe/no/Norway-ortho2.geojson:12]: Hm, you are right, should be 19 of course. Thank you!

**Pull request merged**